packages/core/test/financial-templates/expiring-multiparty/PricelessPositionManager.js

```diff
@@ -1746,9 +1746,19 @@ contract("PricelessPositionManager", function(accounts) {
     await store.setFixedOracleFeePerSecondPerPfc({ rawValue: toWei("0.01") });
 
     // Move time in the contract forward by 1 second to capture a 1% fee.
+
     const startTime = await pricelessPositionManager.getCurrentTime();
     await pricelessPositionManager.setCurrentTime(startTime.addn(1));
 
+    // getCollateral for a given sponsor should correctly reflect the pending regular fee that has not yet been paid.
+    // As no function calls have been made after incrementing time, the fees are still in a "pending" state.
+    // Sponsor has a position with 1e18 collateral in it. After a 1% fee is applied they should have 0.99e18.
+    assert.equal((await pricelessPositionManager.getCollateral(sponsor)).toString(), toWei("0.99"));
```

Comment:
this test verifies that a pending regular fee is applied to the collateral and total position collateral.